src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -128,7 +138,7 @@
             <th>{{ _('Currently on AMO') }}</th>
             <th>{{ _('Status') }}</th>
             <th>{{ _('Validation') }}</th>
-            <th class="version-delete">{{ _('Delete') }}</th>
+            <th class="version-delete">{{ _('Delete/Disable') }}</th>
```

Comment:
There's a button that does two things? That's confusing. The wording seems very unclear to me. Which does it do? Delete sounds VERY permanent and would make me afraid to click this.